File: benchmarks/11_sso_bench.py
import time

def bench_sso():
    print("=== PYTHON SSO TEST ===")
    start = time.time()
    count = 0
    for i in range(1000000):
        s1 = "id" + str(i)
        if s1 == "id500000":
            count += 1
    end = time.time()
    print(f"Result: {(end - start) * 1000:.2f} ms")

bench_sso()

File: benchmarks/11_sso_bench.lua
function bench_sso()
    print("=== LUAJIT SSO TEST ===")
    local start = os.clock() * 1000
    local count = 0
    for i = 0, 999999 do
        local s1 = "id" .. i
        if s1 == "id500000" then
            count = count + 1
        end
    end
    local end_t = os.clock() * 1000
    print(string.format("Result: %.2f ms", end_t - start))
end

bench_sso()
